src/libraries/System.Net.Sockets/tests/FunctionalTests/ArgumentValidationTests.cs

```diff
@@ -800,6 +800,11 @@ public void Connect_ConnectTwice_NotSupported(int invalidatingAction)
         {
             using (Socket client = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp))
             {
+                // On Qemu we omit one test case due to: https://gitlab.com/qemu-project/qemu/-/issues/2410
+                if (!PlatformDetection.IsNotQemuLinux && invalidatingAction == 1)
```

Comment:
```suggestion
                if (PlatformDetection.IsQemuLinux && invalidatingAction == 1)
```
Avoid the double-negation